lib/modules/selectedEntry.js

```diff
@@ -144,6 +144,11 @@ export function select(thing, options) {
 	_select(thing, options);
 }
 
+export const selectClosestVisible = _.debounce(() => {
```

Comment:
Cool, have removed the debounce and added the _select options as a parameter to be consistent with the rest of the select functions